src/main/java/uk/gov/hmcts/ccd/data/casedetails/query/CaseDetailsQueryBuilder.java

```diff
@@ -121,11 +121,13 @@ public CaseDetailsQueryBuilder whereMetadata(MetaData metadata) {
         return this;
     }
 
-    public CaseDetailsQueryBuilder orderByCreatedDate(String sortDirection) {
+    public CaseDetailsQueryBuilder orderBy(MetaData metadata) {
+        String sortDirection = metadata.getSortDirection().orElse("asc");
+        String sortField = metadata.getSortField().orElse(CREATED_DATE);
         if (sortDirection.equalsIgnoreCase("asc")) {
```

Comment:
default sorting may be more useful to be in reverse chronological order for date / time fields. but not an issue at this time.

Reply:
agree, although I just retained existing behaviour.  Also DIVORCE asked specifically for oldest first (ascending).